assets/agenda/components/LabelGroup.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+interface IProps {
+    orientation?: 'horizontal' | 'vertical'; // defaults to 'horizontal'
+    justify?: 'start' | 'end' | 'center'; // defaults to 'start'
+    align?: 'start' | 'end' | 'center'; // defaults to 'center'
+    children: React.ReactNode;
+    className?: string;
```

Comment:
imo this is something we want to avoid in general

Reply:
can u be more specific? :D

Reply:
now is clear